jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/list/AbstractMultiGridViewTest.java

```diff
@@ -199,4 +210,28 @@ public void testGlobalPreferences() {
         getView().loadListTable("key",
                                 consumer);
     }
+
+    @Test
+    public void testControlBulkOperationsEnabled() {
+
+        ListTable extendedPagedTable = mock(ListTable.class);
```

Comment:
@nmirasch could you please move the mock declarations to the beginning of the class (using the annotations) ?